	<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
	<html>
	<head>
	<title>NGS QC Toolkit</title>
	<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
	
	
	<style>
		BODY {
			margin-top: 0;
			background-repeat: repeat;
			font-family: Arial, Helvetica, sans-serif;
			font-size: 5px;
			margin-bottom: 0;
		}
		.cnt {
			font-family: Verdana, Arial, Helvetica, sans-serif;
			font-size: 12px;
			line-height: 20px;
			padding: 5px 20px 0px;
		}
		Table .exp {
			font-family: Verdana, Arial, Helvetica, sans-serif;
			font-size: 13px;
			line-height: 20px;
		}
		TD .padding {
			padding: 0px 20px;
		}
		.head1 {
			font-size: 18px;
			font-weight: bold;
			padding: 5px 0px;
		}
		.head2 {
			font-size: 14px;
			font-weight: bold;
			padding: 5px 0px;
		}
		.head3 {
			font-size: 12px;
			font-weight: bold;
		}
		A {
			text-decoration: none;
			color: #0000FF;
		}
		.tblBg TABLE TD {
			background-color: #EEEEEE;
		}
		.tblBg2 TABLE TD {
			background-color: #E1E1E1;
		}
	</style>
	
	</head>
	
	<body bgcolor="#bbbbbb">
	<table align="center" width="900" cellspacing="0" cellpadding="0" bgcolor="#ffffff" border="0">
	
		<tr>
			<td width="17" rowspan="6">&nbsp;</td>
			<td height="150" bgcolor="#E1E1E1"><center><a href="http://www.nipgr.res.in/ngsqctoolkit.html"><b><font  style="font-size: 50px;">NGS QC T</font><font style="font-size: 40px;">OOLKIT</font></b></center></a>
			</td>
			<td width="17" rowspan="6"></td>
		</tr>
		<tr>
			<td valign="top" class="tblBg">
			<div class="cnt">
				<table class="cnt" width="100%" border="0">
					<tr>
						<td class="head1">Results of quality control (QC) using 454QC.pl v2.3
 <font style="font-size:10px">(Mon Dec 21, 2015)</font></td>
					</tr>
					<tr>
						<td class="head2">Input files and parameters:</td>
					</tr>
					<tr><td class="tblBg2">
						<table width="100%" border="0" class="cnt">
						<tr>
							<td>Analysis type</td><td>Quality check and filtering of 454 data</td>
						</tr>
						<tr>
							<td>Input file directory</td><td><a href="file:///home/genomica/Data/302/" target="_blank">/home/genomica/Data/302/</a></td>
						</tr>
						<tr>
							<td>Input sequence file</td><td>302_in.454.fasta</td>
						</tr>
						<tr>
							<td>Input quality file</td><td>302_in.454.fasta.qual</td>
						</tr>
						<tr>
							<td>Input file format</td><td>454 format (FASTA and QUAL)</td>
						</tr>
						<tr>
							<td>Primer/Adaptor library</td><td>Rapid Library (Standard)
</td>
						</tr>
						<tr>
							<td>Homopolymer trimming</td><td>Off
</td>
						</tr>
						<tr>
							<td>Length of the homopolymer to be removed</td><td>NA</td>
						</tr>
						<tr>
							<td>Length filter</td><td>On
</td>
						</tr>
						<tr>
							<td>Cut-off for minimum read length</td><td>100
</td>
						</tr>
						<tr>
							<td>Cut-off read length for HQ</td><td>70%
</td>
						</tr>
						<tr>
							<td>Cut-off quality score</td><td>20
</td>
						</tr>
						<tr>
							<td>Only statistics</td><td>Off</td>
						</tr>
						<tr>

							<td>Number of processes</td><td>1
</td>

						</tr>
						</table>
					</td>
					</tr>
					<tr>
						<td class="head2">Output files:</td>
					</tr>
					<tr><td class="tblBg2">
						<table width="100%" border="0" class="cnt">
						<tr>
							<td>Output folder</td><td><a href="file:///home/genomica/Data/302/454QC_Filtered_files/" target="_blank">/home/genomica/Data/302/454QC_Filtered_files/</a></td>
						</tr>
						<tr>
							<td>QC statistics</td><td><a href="302_in.454.fasta_stat" target="_blank">302_in.454.fasta_stat</a></td>
						</tr>
						<tr>
							<td>High quality filtered sequence file</td><td>302_in.454.fasta_filtered</td>
						</tr>
						<tr>
							<td>High quality filtered quality file</td><td>302_in.454.fasta.qual_filtered</td>
						</tr>
						<tr>
							<td>Length distribution</td><td><a href="302_in.454.fasta_lenDistribution.png" target="_blank">302_in.454.fasta_lenDistribution.png</a></td>
						</tr>
						<tr>
							<td>Base composition</td><td><a href="302_in.454.fasta_baseCompostion.png" target="_blank">302_in.454.fasta_baseCompostion.png</a></td>
						</tr>
						<tr>
							<td>GC content distribution</td><td><a href="302_in.454.fasta_gcDistribution.png" target="_blank">302_in.454.fasta_gcDistribution.png</a></td>
						</tr>
						<tr>
							<td>Quality distribution</td><td><a href="302_in.454.fasta_qualDistribution.png" target="_blank">302_in.454.fasta_qualDistribution.png</a></td>
						</tr>
						<tr>
							<td>Summary of QC</td><td><a href="302_in.454.fasta_summary.png" target="_blank">302_in.454.fasta_summary.png</a></td>
						</tr>
						</table>
					</td>
					</tr>
					<tr>
						<td class="head2" style="background-color: #ffffff;">&nbsp;</td>
					</tr>
					<tr>
						<td class="head2">Results of QC</td>
					</tr>
<tr><td class="head3">QC statistics</td></tr>
<tr><td class="tblBg2"><table width="100%" border="0" class="cnt">
<tr><td>File name</td><td>302_in.454.fasta</td></tr>
<tr><td>Total number of reads</td><td>799</td></tr>
<tr><td>Total number of trimmed reads containing homopolymer</td><td>0</td></tr>
<tr><td>Total number of trashed reads (<100 bp in length after trimming)</td><td>2</td></tr>
<tr><td>Total number of low quality reads (excluding <100 reads)</td><td>0</td></tr>
<tr><td>Total number of HQ reads</td><td>797</td></tr>
<tr><td>Percentage of HQ reads</td><td>99.75%</td></tr>
<tr><td>Total number of bases</td><td>413118</td></tr>
<tr><td>Total number of bases in HQ reads</td><td>412959</td></tr>
<tr><td>Total number of HQ bases in HQ reads</td><td>342837</td></tr>
<tr><td>Percentage of HQ bases in HQ reads</td><td>83.02%</td></tr>
<tr><td>Number of Primer/Adaptor trimmed reads</td><td>257</td></tr>
<tr><td>Total number of HQ filtered reads</td><td>797</td></tr>
<tr><td>Percentage of HQ filtered reads</td><td>99.75%</td></tr>
</table></td></tr>
<tr><td class="head3">Detailed QC statistics</td></tr>
<tr><td class="tblBg2"><table width="100%" border="0" class="cnt">
<tr><td>File name</td><td>302_in.454.fasta</td><td>302_in.454.fasta_filtered</td></tr>
<tr><td>Total number of reads</td><td>799</td><td>797</td></tr>
<tr><td>Minimum read length</td><td>65</td><td>269</td></tr>
<tr><td>Maximum read length</td><td>602</td><td>602</td></tr>
<tr><td>Average read length</td><td>517.04</td><td>508.61</td></tr>
<tr><td>Median read length</td><td>522</td><td>517</td></tr>
<tr><td>N25 length</td><td>544</td><td>542</td></tr>
<tr><td>N50 length</td><td>524</td><td>521</td></tr>
<tr><td>N75 length</td><td>499</td><td>487</td></tr>
<tr><td>N90 length</td><td>472</td><td>447</td></tr>
<tr><td>N95 length</td><td>454</td><td>424</td></tr>
<tr><td>Total number of bases</td><td>413118</td><td>405360</td></tr>
<tr><td>Total number of HQ bases</td><td>342972</td><td>337134</td></tr>
<tr><td>Percentage of HQ bases</td><td>83.02%</td><td>83.17%</td></tr>
<tr><td>Average quality score (Overall)</td><td>41.99</td><td>42.15</td></tr>
</table></td></tr>
					<tr>
						<td class="head3">Summary of QC</td>
					</tr>
					<tr>
						<td>
							<table width="100%" border="0" class="cnt">
								<tr>
									<td><div align="justify">Following pie chart shows the summary of QC depicting percentage of high quality, trimmed (homopolymer and/or contamination) and trashed (low quality and/or short) reads.</div></td>
								</tr>
								<tr>
									<td align="center"><img src="302_in.454.fasta_summary.png" border="1"><br><b>(A)</b></td>
								</tr>
							</table>
						</td>
					</tr>
					<tr>
						<td class="head3">Length distribution</td>
					</tr>
					<tr>
						<td>
							<table width="100%" border="0" class="cnt">
								<tr>
									<td><div align="justify">Following graph(s) show number of reads for sequence length range for input file,  302_in.454.fasta(A), before and after QC.</div></td>
								</tr>
								<tr>
									<td align="center"><img src="302_in.454.fasta_lenDistribution.png" border="1"><br><b>(A)</b>
								</tr>
							</table>
						</td>
					</tr>
					<tr>
						<td class="head3">GC content distribution</td>
					</tr>
					<tr>
						<td>
							<table width="100%" border="0" class="cnt">
								<tr>
									<td><div align="justify">Following graph(s) show number of reads for distinct average GC content (%) ranges for input file,  302_in.454.fasta(A), before and after QC.</div></td>
								</tr>
								<tr>
									<td align="center"><img src="302_in.454.fasta_gcDistribution.png" border="1"><br><b>(A)</b>
								</tr>
							</table>
						</td>
					</tr>
					<tr>
						<td class="head3">Quality distribution</td>
					</tr>
					<tr>
						<td>
							<table width="100%" border="0" class="cnt">
								<tr>
									<td><div align="justify">Following graph(s) show number of reads for different average PHRED quality scores for input file,  302_in.454.fasta(A), before and after QC.</div></td>
								</tr>
								<tr>
									<td align="center"><img src="302_in.454.fasta_qualDistribution.png" border="1"><br><b>(A)</b>
								</tr>
							</table>
						</td>
					</tr>
					<tr>
						<td class="head3">Base composition</td>
					</tr>
					<tr>
						<td>
							<table width="100%" border="0" class="cnt">
								<tr>
									<td><div align="justify">Following graph(s) show base composition (count) for input file,  302_in.454.fasta(A), before and after QC with percentage of bases at the bottom.</div></td>
								</tr>
								<tr>
									<td align="center"><img src="302_in.454.fasta_baseCompostion.png" border="1"><br><b>(A)</b>
								</tr>
							</table>
						</td>
					</tr>
				</table>
			</div>
			</td>
		</tr>
		<tr><td style="font-size: 11px;" align="center" valign="middle" height="25" background="/home/genomica/Programs/NGSQCToolkit_v2.3/QC/lib/imgs/btmLine.png" bgcolor="#EEEEEE">For Questions and Suggestions, contact <a href="mailto:mjain@nipgr.res.in">Mukesh Jain (mjain@nipgr.res.in)</a>; <a href="mailto:ravipatel@nipgr.res.in">Ravi Patel (ravipatel@nipgr.res.in)</a></td></tr>
	
		<!--- <tr><td colspan="3"><img src="/home/genomica/Programs/NGSQCToolkit_v2.3/QC/lib/imgs/down.png"></td></tr> --->
	</table>
	</body>
	</html>
